providers/provider.num

```diff
@@ -0,0 +1 @@
+OSSL_provider_init                     1	*	EXIST::FUNCTION:
```

Comment:
Wait... Isn't there one in util already?

Reply:
It looks like it, in `util/providers.num`